app/src/main/java/com/nmc/android/ui/LauncherActivity.kt

```diff
@@ -41,6 +43,25 @@ class LauncherActivity : BaseActivity() {
     @Inject
     lateinit var appPreferences: AppPreferences
 
+    private val handler = Handler(Looper.getMainLooper())
```

Comment:
Rebase and resolve the conflict for privacy once this upstremead PR (https://github.com/nextcloud/android/pull/11760) is merged by NC.
Applicable for onboarding and splash PR as well.